server_benchmark.py

```diff
@@ -113,12 +113,15 @@ def benchmarkOneQuery(request, repeatLimit=3, pageLimit=3):
     # return sum(times[2:])/len(times[2:])
     return min(times)
 
-if __name__ == '__main__':
+
+def main(args):
+    global __BACKEND__
```

Comment:
Global variables can lead to nasty headaches, and don't scale well in terms of code complexity. It would be a bit nicer if we could use a class to encapsulate the state (the QueryBenchmarker class suggested in the issue). This would have an instance variable ``self._backend``, which keeps this information.